force-app/main/default/triggers/AccountTrigger.trigger

```diff
@@ -0,0 +1,9 @@
+trigger AccountTrigger on Account (before insert) {
+    for (Account acc : Trigger.new) {
+        // Check if the Account Type is empty
+        if (String.isBlank(acc.Type)) {
+            // Set the Account Type to 'Prospect'
+            acc.Type = 'Prospect';
+        }
+    }
+}
```

Comment:
The logic here is correct but you never want more than one trigger on an SObject ever.  I would try and refactor this later to one Account Trigger.